components/layout_2020/flow/inline.rs

```diff
@@ -619,13 +619,15 @@ impl<'a, 'b> InlineFormattingContextState<'a, 'b> {
         );
 
         if inline_box.is_first_fragment {
-            self.current_line.inline_position += inline_box_state.pbm.padding.inline_start +
+            self.current_line.inline_position += (inline_box_state.pbm.padding.inline_start +
                 inline_box_state.pbm.border.inline_start +
                 inline_box_state
                     .pbm
                     .margin
                     .inline_start
-                    .auto_is(Length::zero);
+                    .auto_is(Length::zero)
+                    .into())
+            .into();
```

Comment:
Converting a Length back and forth seems a bit weird. Consider
```rs
            self.current_line.inline_position += Length::from(
                inline_box_state.pbm.padding.inline_start +
                    inline_box_state.pbm.border.inline_start,
            ) + inline_box_state
                .pbm
                .margin
                .inline_start
                .auto_is(Length::zero);
```

Reply:
Thanks, updated